src/server/models/eGauge/eGaugeRequestor.js

```diff
@@ -19,7 +19,11 @@ class EgaugeRequestor {
 		// Information expected by the eGauge API and needed by OED.
 		const url = new URL(`https://${meter.url}`);
 		this.apiUrl = `${url.origin}/api`; // can we add an ending slash??
-		this.username = url.searchParams.get('username');
+		// allowing for backwards compatibility if previous eGuage meters are using the 'email' parameter instead of
```

Comment:
A couple of thoughts. First, TODO per other comments. Second, issue if provide both to settle.

Reply:
works the same as the others, takes username if it exists, otherwise takes email. If neither, then fails